packages/in_app_purchase/in_app_purchase/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
-## NEXT
+## 2.0.0
 
 * Fixes integration tests.
 * Updates example app Android compileSdkVersion to 31.
```

Comment:
Nit: these two lines should probably go to the end since they are not important to clients. We could swap these with the breaking changes section so that's first (rather than having a sublist in the middle of the list).